documentation/faq/collections.md

```diff
@@ -0,0 +1,41 @@
+---
+layout: page
+title: Preparation for Converting Your Role to the Collections Format
+---
+
+If you are developing a role in linux-system-roles, you may find your question already answered here.
+
+## Module and Module_util name
+
+<b>Q:</b> My role has a custom module in the `library/` directory. Are there anything I should know of?<br>
+<b>A:</b> All of the files in the `library/` directory should have a `YOUR_ROLENAME_` prefix. For instance, if you are planning to name your module `getinfo`, please name it `YOUR_ROLENAME_getinfo`. This will help preventing the module name conflict when your role is converted to the collections format. In the format, all the modules are placed in the same directory `ansible_collections/NAMESPACE/COLLECTION_NAME/plugins/modules/`, where popular names could get conflicted.
+
+<b>Q:</b> How about module_utils?<br>
+<b>A:</b> The `module_utils/` directory in the collections format is allowed to have sub-directories. Please put all the files in your `module_utils/` directory in `module_utils/YOUR_ROLENAME` or `module_utils/YOUR_ROLENAME_lsr/` sub-directory.
```

Comment:
only `module_utils/YOUR_ROLENAME_lsr/` - best to completely avoid any chance of conflict